runtime/rasdump/TextFileStream.cpp

```diff
@@ -90,6 +100,11 @@ bool TextFileStream::isOpen(void) const
 	return _FileHandle != -1;
 }
 
+bool TextFileStream::isSTDOUTERR(void) const
+{
+	return (!_IsOpen && (_FileHandle != -1));/* Returns true if writting to STDOUT/ERR*/
```

Comment:
Please remove the outer-most parentheses, and fix the typo: writing has only one 't'.